2D_Array/readme.md

```diff
@@ -15,5 +15,7 @@ Due to the fact that the elements of 2D arrays can be random accessed. Similar t
 * Row-wise sum ----> [C++](/Code/C++/row_wise_sum.cpp)
 * Spiral Print ----> [C++](/Code/C++/spiral_print.cpp)
 * Staircase Search ----> [C++](/Code/C++/staircase_search.cpp)
+* Transpose of a Matrix --->[Java](/Code/Java/transpose.java)
```

Comment:
Instead of these 2 same line issue Combine them like this
* Transpose of a Matrix --->[Java](/Code/Java/transpose.java) | [Python](/Code/Python/Transpose_of_matrix.py)